### ISSUE TITLE
#popper slot is not available

### ISSUE THREAD

**Issue by sdr0x07b6** [Akryum/floating-vue#1046]
This is my first time using floating-vue.
Since I use Nuxt 3, I installed it by using `floating-vue/nuxt` as per the installation guide.

```json
"dependencies": {
  "floating-vue": "^5.2.2",
```

```ts
modules: [
  'floating-vue/nuxt',
],
```

Next I wrote a template following the example in the documentation.
However, I get an error for the `popper` slot of `VTooltip` and `VMenu`.

```
Property 'popper' does not exist on type 'Readonly<{ default?: ((arg: { isActive: Ref<boolean>; }) => VNode<RendererNode, RendererElement, { [key: string]: any; }>[]) | undefined; activator?: ((arg: { ...; }) => VNode<...>[]) | undefined; }>'. ts-plugin(2339)
```

![popper](https://github.com/user-attachments/assets/efe5951f-924f-4569-b5ba-208c07c13d35)

All I've done is install it and write sample markup, so I don't know what to fix.
What should I change?

```
% yarn nuxi info

------------------------------
- Operating System: Darwin
- Node Version:     v20.12.1
- Nuxt Version:     3.12.4
- CLI Version:      3.12.0
- Nitro Version:    2.9.7
- Package Manager:  yarn@1.22.22
- Builder:          -
- User Config:      devtools, srcDir, modules, vuetify
- Runtime Modules:  vuetify-nuxt-module@0.16.1, floating-vue/nuxt
- Build Modules:    -
------------------------------
```

**Comment by sdr0x07b6** [comment]
The above report briefly describes the installation status.
In fact, the vuetify-nuxt-module is also installed.

It turns out that the issue can be reproduced just by installing this.

```json
"dependencies": {
  "floating-vue": "^5.2.2",
  },
"devDependencies": {
  "vuetify-nuxt-module": "^0.16.1"
```

```ts
modules: [
  'vuetify-nuxt-module',
  'floating-vue/nuxt',
],
```

I prepared Stackblitz.
https://stackblitz.com/edit/nuxt-starter-ldfg7v?file=package.json,nuxt.config.ts,app.vue
All I did was introduce `vuetify-nuxt-module`, place `floating-vue` component in the template (`App.vue`), and implement `#popper` slot.
As a result, `VTooltip` and `VMenu` are not displayed.

In VSCode I get the following error:
```
Property 'popper' does not exist on type 'Readonly<{ default?: ((arg: { isActive: Ref<boolean>; }) => VNode<RendererNode, RendererElement, { [key: string]: any; }>[]) | undefined; activator?: ((arg: { ...; }) => VNode<...>[]) | undefined; }>'. ts-plugin(2339)
```

If I exclude `vuetify-nuxt-module` from `modules` , `VTooltip` and `VMenu` are displayed normally.
Static analysis errors in VSCode also disappear.

Can this be resolved without either `vuetify-nuxt-module` or `floating-vue/nuxt` fixing the conflict? Is there anything I can do on my side?

**Comment by sdr0x07b6** [comment]
I asked about this in the Issue on the vuetify-nuxt-module.
https://github.com/userquin/vuetify-nuxt-module/issues/249

I was kindly told that the problem was caused by a name conflict between the floating-vue component and the Vuetify component. `v-tooltip` `v-menu`

Now that you mention it, that makes sense...!

I solved the problem by implementing a plugin myself that would be installed to avoid name conflicts, without using the `floating-vue/nuxt` module.
